tests/core/engine_adapter/test_integration.py

```diff
@@ -872,3 +881,74 @@ def test_scd_type_2(ctx: TestContext):
             ]
         ),
     )
+
+
+def test_sushi(ctx: TestContext):
+    if ctx.test_type == "df":
+        pytest.skip("Sushi end-to-end tests only need to run for query and pyspark tests")
+
+    # this skips the pyspark test if the engine isn't pyspark-enabled
+    ctx.init()
```

Comment:
This is my bad that I coupled the pyspark check with init since you don't actually need the rest of init. I can clean that up in follow up PR and then allow this test to skip doing the full init when it doesn't need it.